timeless/auth/auth.py

```diff
@@ -4,17 +4,16 @@
 
 def login(username="", password=""):
     """Login user
-    @todo #59:30min The backend should return only error codes for login procedure,
-     not hardcoded messages. The codes should then be translated into (mapped to)
-     Strings on the UI, so we can easily have internationalization or change
-     error messages without having to rebuild + redeploy the backend.
+    @todo #201:30min Translate login error codes.
+      The codes returned from login errors should then be translated into (
+      mapped to) Strings on the UI, so we can easily have
+      internationalization or change error messages without having to
+      rebuild + redeploy the backend.
```

Comment:
@paulodamaso I would say this puzzle is not needed because all that mapping logic will be in the React Client. I'd say there is nothing more to be done on the backend.

Reply:
@emilianodellacasa Wont' we implement the client too?

Reply:
@paulodamaso Of course

Reply:
@emilianodellacasa So the puzzle stays?

Reply:
@paulodamaso IMHO the puzzle stays, we agreed we will develop part react (for complex pages) and common templates for simple ones; login page definitely is a simple page;